app_flutter/lib/service/fake_cocoon.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:fixnum/fixnum.dart';
+
+import 'package:cocoon_service/protos.dart'
+    show Commit, CommitStatus, Stage, Task;
+
+import 'cocoon.dart';
+
+class FakeCocoonService implements CocoonService {
+  @override
+  Future<List<CommitStatus>> getStats() {
+    return Future.delayed(Duration(seconds: 1), () => _getFakeStats());
+  }
+
+  List<CommitStatus> _getFakeStats() {
+    List<CommitStatus> stats = List();
+
+    final int baseTimestamp = DateTime.now().millisecondsSinceEpoch;
+
+    for (int i = 0; i < 100; i++) {
+      Commit commit = _getFakeCommit(i, baseTimestamp);
+
+      CommitStatus status = CommitStatus()
+        ..commit = commit
+        ..stages.addAll(_getFakeStages(i, commit));
+
+      stats.add(status);
+    }
+
+    return stats;
+  }
+
+  Commit _getFakeCommit(int index, int baseTimestamp) {
```

Comment:
I wouldn't name these methods with `get` because they are actually instantiating the Commit object:

```suggestion
  Commit _createFakeCommit(int index, int baseTimestamp) {
```

Reply:
That's a good point, thank you for sharing. Done!